website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -736,7 +763,7 @@ process, and should be displayed as such when reporting results.
 | s3_last_update        | `timestamptz` | LastModified key value from the S3 listing. | NULLABLE column |
 | orca_size             | `int8`    | Size value from the ORCA File object the file belongs to. | From Catalog. |
 | s3_size               | `int8`    | Size key value from the S3 listing.       | NULLABLE column |
```

Comment:
s3_size shows as NULLABLE but is NOT NULL in the architecture diagram

Reply:
Oof, missed a lot of those. Fixed.